source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRContentStackView.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ACRContentStackView
+//  ACRContentStackView.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ACRContentStackView: UIView
```

Comment:
>ACRContentStackView [](start = 11, length = 19)

It looks like it might be simpler to have this interface derive from UIStackView rather than containing a UIStackView as a subview.